src/java.base/share/classes/java/util/ImmutableCollections.java

```diff
@@ -878,12 +873,65 @@ public List<E> reversed() {
             return new StableReverseOrderListView<>(this);
         }
 
+        @Override
+        public List<E> subList(int fromIndex, int toIndex) {
+            subListRangeCheck(fromIndex, toIndex, size());
+            return StableSubList.fromStableList(this, fromIndex, toIndex);
+        }
+
         @Override
         public String toString() {
             return StableUtil.renderElements(this, "StableList", delegates);
```

Comment:
```suggestion
            return StableUtil.renderElements(this, "StableCollection", delegates);
```
I think `StableList`, `StableSubList`, `StableReverseOrderListView`, `StableMapEntrySet` and `StableMapValues` should all use `"StableCollection"` to be consistent with `AbstractCollection`'s use of `"Collection"`. `StableMap` should use `"StableMap"` to be consistent with `AbstractMap`'s use of `"Map"`.